types/Configuration.d.ts

```diff
@@ -86,6 +86,54 @@ declare namespace OIPF {
          */
         pvrEndPadding: number;
 
+        /**
+         * Shall be set to false if subtitles are disabled by the terminal. When set to false, subtitle components that
+         * are selected using a video/broadcast object, A/V control object or HTML5 media element will not be presented.
+         * See also clause 10.2.7 in HBBTV Spec
+         */
+        subtitlesEnabled: boolean;
```

Comment:
According the spec these are read-only properties. I can't see a downside off-hand to reflecting that here - if you can't either, could you add that?

Reply:
Yeah i thought about that but none of the others had it so didnt include. Can add to others aswell?

Reply:
I had a quick look at the OIPF spec, and it _doesn't_ document them as read-only, even though I bet they are.
I also feel a little disquieted about the inconsistency between the two sets and hesitated before leaving the comment.
My feeling was we should just reflect what the docs say. I think we can either make these new props readonly, or not, but leave the others as they are.
Happy with whatever you decide to do with that.